src/routes.js

```diff
@@ -0,0 +1,12 @@
+const routes = {
+  HOME: '/',
+  ABOUTUS: '/about-us',
+  HOWITWORKS: '/how-it-works',
+  SIGNUP: '/signup',
+  LOGIN: '/login',
+  LOGOUT: '/logout',
+  DONATION: '/donation',
+  REQUEST: '/request',
+};
+
+export default routes;
```

Comment:
Lets export them one by one, so we can just import one thing and not waste any memory. Also, for the file names with more than one word, use "_" between the words, ex: ABOUT_US, HOW_IT_WORKS etc.